syft/frameworks/torch/hook/hook.py

```diff
@@ -1134,3 +1135,47 @@ def optim_float_precision_(optim_self):
             return optim_self
 
         self.torch.optim.Optimizer.float_precision = optim_float_precision_
+
+        # Modification of torch/nn/utils/clip_grad.py
```

Comment:
Can you make more explicit in the comment why do we need a specific function for clipping remote tensors? maybe add comments to the lines that were changed and why/how they were changed?

Reply:
Made the comment at the beginning of the function more explicit about the motivation for having this extra function. Actually, the only two lines changed are the following ones:

```
total_norm = torch.zeros(1)
#Let's send the total norm over to
total_norm = total_norm.send(worker)
```